app/src/main/java/com/yml/womensafety/repository/EscapeThreatRepository.kt

```diff
@@ -0,0 +1,58 @@
+package com.yml.womensafety.repository
+
+import android.content.Context
+import androidx.lifecycle.MutableLiveData
+import com.yml.womensafety.R
+import com.yml.womensafety.navigationdrawer.LawsAndEscapeThreat
+
+class EscapeThreatRepository(val context: Context) {
+    private var escapeThreatDataSet = ArrayList<LawsAndEscapeThreat>()
+
+    fun getEscapeThreatData(): MutableLiveData<List<LawsAndEscapeThreat>> {
+        displaySituationsAndSolutions()
+        val escapeThreatData: MutableLiveData<List<LawsAndEscapeThreat>> = MutableLiveData()
```

Comment:
As per the MVVM arch, the livedata should be present in the view model layer and not is repo. We shoudl only have network calls and db operations in the repo layer